tests/test_io.py

```diff
@@ -23,28 +26,115 @@ def captured_output():
 
 
 class Test_load_program_texts():
-    def test_load_program_texts_01(self):
+     def test_load_program_texts_in_global_PROGTEXT(self):
+         with mock.patch("builtins.open", mock.mock_open(read_data="[header]\r\ncontent\r\n")) as mock_file:
+            dfastmi.io.load_program_texts("")
+            assert dfastmi.io.PROGTEXTS['header'] == ['content']
+         
+     def test_load_program_texts_multiline_in_global_PROGTEXT(self):
+         with mock.patch("builtins.open", mock.mock_open(read_data="[header]\r\ncontent\r\ncontent2\r\n")) as mock_file:
+            dfastmi.io.load_program_texts("")
+            assert dfastmi.io.PROGTEXTS['header'] == ['content', 'content2']
+         
+     def test_load_program_texts_line_header_with_no_value_in_global_PROGTEXT(self):
+         with mock.patch("builtins.open", mock.mock_open(read_data="[header]\r\n[otherheader]\r\ncontent\r\n")) as mock_file:
+            dfastmi.io.load_program_texts("")
+            assert dfastmi.io.PROGTEXTS['header'] == []
+            assert dfastmi.io.PROGTEXTS['otherheader'] == ['content']
+         
+     def test_load_program_texts_double_header_throws_exception(self):
+         with mock.patch("builtins.open", mock.mock_open(read_data="[header]\r\n[header]\r\ncontent\r\n")) as mock_file:
+            with pytest.raises(Exception) as cm:
+                dfastmi.io.load_program_texts("")
+            assert str(cm.value) == 'Duplicate entry for "header" in "".'
+         
+    
+class Test_data_access_load_program_texts():
+    def test_load_program_texts_load_default_uk_messages_file(self):
```

Comment:
Should we add the same test for NL?

Reply:
As discussed with @MRVermeulenDeltares, this should be resolved in [DFAST-107](https://issuetracker.deltares.nl/browse/DFAST-107)